lib/npm-auditer.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/* eslint-disable max-len */
+const childProcess = require('child_process');
+
+function reportAudit(report, config) {
+  return new Promise((resolve, reject) => {
```

Comment:
Unfortunately, the diff on this file is not a good one. This is because I also removed the unnecessary `npm-audit-report` dependency.